### DISCUSSION TITLE
Does Livewire's File Upload temporary upload use s3's multipart upload feature?

### DISCUSSION THREAD

**Discussion by sawirricardo** [livewire/livewire#5917] [1. Help]
Currently, the source code only uses the s3 upload features, which can be a bottleneck if working with big files > 5gb

Reference: https://docs.aws.amazon.com/AmazonS3/latest/userguide/mpuoverview.html

**Comment by joshhanley** [comment]
@sawirricardo yeah I don't think it does use multipart upload.

**Reply by sawirricardo** [reply]
Ill make a pr for this

**Reply by sawirricardo** [reply]
https://github.com/livewire/livewire/pull/5959 PR will go here

**Comment by votkapower** [comment]
Still needed and very hard to do manualy, can we reopen this PR?
